### PR TITLE
ci: 👷 stop major updates from automerging

### LINKED ISSUE #497: ci: 👷 stop major updates from automerging
## User Story
As a user of the Solid Design System, I would like to make sure major updates are not merged without a review
so that breaking changes don't get into the codebase.
## DoR
- [ ] Item has business value
- [ ] Item has been estimated by the team
- [ ] Item is clear and well-defined
- [ ] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Implementation works successfully on `feature` branch

### PR DESCRIPTION
**Pull request by Vahid1919**
## Description:
Removed redundant rule to update dev dependencies that also auto-merged major updates, added blanked rule to turn off automerge for major updates. Closes  #497

## Definition of Reviewable:
*PR notes: Irrelevant elements should be removed.*
- [x] relevant tickets are linked
- [x] PR is assigned to project board

### CODE REVIEW

**Comment by karlbaumhauer** [comment]
@Vahid1919 LGTM. We might just wait for @mariohamann to finish his fix on the pipelines to include this into your feature branch.

**Review by yoezlem — APPROVED** [review]

File: renovate.json [hunk 1]
```diff
@@ -6,11 +6,7 @@
   "packageRules": [
     {
       "matchUpdateTypes": ["minor", "patch"],
-      "matchCurrentVersion": "!/^0/",
-      "automerge": true
-    },
-    {
-      "matchDepTypes": ["devDependencies"],
+      "matchCurrentVersion": ">= 1.0.0",
```
Comment (mariohamann) [inline renovate.json:9]: I don't think that this makes sense. This only works for specific packages.

**Review by mariohamann — CHANGES_REQUESTED** [review]

File: renovate.json (hunk 1 above)
Comment (mariohamann) [reply renovate.json:9]: I think you can remove `matchCurrentVersion` completely.

Comment (mariohamann) [reply renovate.json:9]: NVM. Let's give it a try. :D

**Pull request merged**